api/descriptions/policies/vulnerability_vms_put.md

```diff
@@ -1,33 +1,47 @@
-Updates all rules in your vulnerability policy for VM images in a single shot.
-Updating all rules at the same time makes it possible to maintain strict ordering between rules.
+Updates the policy for your VM images scanned in your pipeline.
+All rules in the policy are updated in a single shot.
+
+This endpoint maps to the policy table in **Defend > Vulnerabilities > INSERT_LOCATION_HERE** in the Console UI.
+
+
+### cURL Request
+
+The following cURL command overwrites all rules in your current policy with a new policy that has a single rule.
+
+```bash
+$ curl 'https://<CONSOLE>/api/v1/policies/vulnerability/vms?project=<PROJECT_NAME>' \
+  -X PUT \
+  -u <USER> \
+  -H 'Content-Type: application/json' \
+  -d \
+'{
+  "rules": [
+    {
+      "name": "<RULE_NAME>",
+      "collections":[
+         {
+            "name":"<COLLECTION_NAME>",
+         }
+      ],
+      "alertThreshold":{
+         "disabled":false,
+         "value":4
+      },
+      "blockThreshold":{
```

Comment:
@KICKdesigns There's no block threshold for VM images rules in the UI. The API might accept it, but it will be meaningless. Please test and validate, then update your curl command.